.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,11 +1,11 @@
 blank_issues_enabled: true
-contact_links: 
-  - name: "Report an issue"
-    about: "For when Collect is behaving in an unexpected way"
-    url: "https://forum.getodk.org/c/support/6"
-  - name: "Request a feature"
-    about: "For when Collect is missing functionality"
-    url: "https://forum.getodk.org/c/features/9"
-  - name: "Everything else"
-    about: "For everything else"
-    url: "https://forum.getodk.org/c/support/6"
\ No newline at end of file
+# contact_links: 
+#   - name: "Report an issue"
+#     about: "For when Collect is behaving in an unexpected way"
+#     url: "https://forum.getodk.org/c/support/6"
+#   - name: "Request a feature"
+#     about: "For when Collect is missing functionality"
+#     url: "https://forum.getodk.org/c/features/9"
+#   - name: "Everything else"
+#     about: "For everything else"
+#     url: "https://forum.getodk.org/c/support/6"
```

Comment:
Hey @AnjumanHasan thanks for the review. You can remove these instead of commenting them out.

Reply:
I've removed the comments!